### ISSUE TITLE
feature request: change bindings?

### ISSUE THREAD

**Issue by thisguychris** [blackCauldron7/surround.nvim#2]
Hi, thanks for making this plugin.

So far, what blocks me, is it's overriding the `s` keybinding, which is being used for `vim-sneak` (I use this plugin a lot). I'm not sure what's the best solution, but `vim-sneak` works well with `vim-surround`.

If perhaps there's an ability to change plugins, or a disable function would do? Having plugins work well together is the preferred choice.

**Comment by blackCauldron7** [comment]
Hmm I can make a setting toggle to change the mappings to match `vim-surround` would that be preferable?

**Comment by thisguychris** [comment]
> Hmm I can make a setting toggle to change the mappings to match `vim-surround` would that be preferable?

a toggle would be preferable 👌 And that this would also open many users too. 🥳

**Comment by blackCauldron7** [comment]
Try the latest commit and see docs on how to change the mappings styles.
TLDR;

let g:surround_mappings_style = "surround"

Please close the issue if the commit works for you.
